scripts/tests/test_synthesize_b0.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from scilpy.io.fetcher import fetch_data, get_home, get_testing_files_dict
```

Comment:
Rename this file and edit this file so it actually tests: `scil_volume_b0_synthesis.py`